types/block.go

```diff
@@ -93,6 +93,11 @@ type (
 		// started being supported).
 		TxnCounter uint64 `codec:"tc"`
```

Comment:
we should add the txn256 commitment here as well.
https://github.com/algorand/go-algorand/blob/7de989aa62a01ce9971b48cb1c2ca6cc2f18f3bf/data/bookkeeping/block.go#L141